### PR TITLE
Out-of-the-box Content Cards Examples

### PR DESCRIPTION
**Pull request by KellieHawks**
Creating out-of-the-box Content Card customization documentation w/ Justin.

### CODE REVIEW

**Comment by KellieHawks** [comment]
Hey @lowip and @hokstuff, when you get a chance, can you quickly review these additions Justin and I made to the iOS content card customization docs. Thanks!

**Review by lowip — APPROVED** [review]
LGTM

File: _docs/_developer_guide/platform_integration_guides/ios/content_cards/customization.md [hunk 1]
```diff
@@ -34,15 +34,165 @@ Note that overriding default images is currently not supported in our Xamarin iO
...
+
+### Providing Custom Interfaces
+
+Custom interfaces can be provided by registering custom classes for each desired card type. 
+
+![Custom Classes]({% image_buster /assets/img/interface1.png %}){: style="max-width:35%;margin-left:15px;"}
+![Custom Classes]({% image_buster /assets/img/interface2.png %}){: style="max-width:25%;margin-left:15px;"}
+![Custom Classes]({% image_buster /assets/img/interface3.png %}){: style="max-width:18%;margin-left:15px;"}
+
+If you would like to use our banner, captioned image, and classic Content Card templates, those can be found [here](). Alternatively, if you would like to provide your own custom interfaces, reference the following code snippets:
```
Comment (hokstuff) [inline _docs/_developer_guide/platform_integration_guides/ios/content_cards/customization.md:126]: `those can be found [here]()`
^ Missing link?

**Review by hokstuff — APPROVED** [review]
LGTM after nit

File: _docs/_developer_guide/platform_integration_guides/ios/content_cards/customization.md [hunk 2]
```diff
@@ -34,15 +34,165 @@ Note that overriding default images is currently not supported in our Xamarin iO
...
+
+### Customizing UI
+
+The following code snippets show how to style and change out-of-the-Box Content Cards to fit your UI needs using methods provided by the SDK. These methods allow you to customize all aspects of the Content Card UI, such as including custom fonts, customized color components, customized text, and more. 
+
+There exist two distinct ways to customize Content Card UI, the dynamic UI method, which allows you to update card UI on a per-card basis, or the static method, which will update the UI across all cards.
```
Comment (lydia-xie) [inline _docs/_developer_guide/platform_integration_guides/ios/content_cards/customization.md:43]: @KellieHawks minor suggestion here! 

"There are two distinct ways to customize the Content Card UI: 
- Dynamic method: allows you to update card UI on a per-card basis
- Static method: updates the UI across all cards"

**Review by lydia-xie — APPROVED** [review]
lgtm!

**Pull request merged**